src/pages/ClinicalHistory.jsx

```diff
@@ -0,0 +1,196 @@
+    },
+  ]);
+
+  const [allergies] = useState(['Peanuts', 'Penicillin']);
+
+  return (
+    <div className="container mx-auto py-8 px-4 text-left">
+    
+      <div className="flex justify-between items-center mb-6">
+        <h1 className="text-3xl font-bold">Patient Clinical History</h1>
+        <Button variant="outline" className="px-4 py-2 text-base">
+          <DownloadIcon className="mr-2 h-5 w-5" />
+          Report
+        </Button>
+      </div>
+      
+      <div className="space-y-6">
+        <Card>
+          <CardHeader>
+            <CardTitle>Conditions</CardTitle>
+            <CardDescription>Current and past medical conditions</CardDescription>
+          </CardHeader>
+          <CardContent>
+            <ul className="space-y-4">
+              {conditions.map((condition, index) => (
+                <li key={index} className="bg-muted p-4 rounded-lg">
+                  <h3 className="font-semibold text-lg">{condition.name}</h3>
+                  <p className="text-sm text-muted-foreground">{condition.details}</p>
+                  <p className="text-sm text-muted-foreground mt-2">Since: {condition.since}</p>
+                </li>
+              ))}
+            </ul>
+          </CardContent>
+        </Card>
+
+        <Card>
+          <CardHeader>
+            <CardTitle>Treatments</CardTitle>
+            <CardDescription>Current and past treatments</CardDescription>
+          </CardHeader>
+          <CardContent>
+            <div className="overflow-x-auto">
+              <Table>
+                <TableHeader>
+                  <TableRow>
+                    <TableHead>Name</TableHead>
+                    <TableHead>Start Date</TableHead>
+                    <TableHead>End Date</TableHead>
+                    <TableHead>Instructions</TableHead>
+                  </TableRow>
+                </TableHeader>
+                <TableBody>
+                  {treatments.map((treatment, index) => (
+                    <TableRow key={index}>
+                      <TableCell className="font-medium">{treatment.name}</TableCell>
+                      <TableCell>{treatment.startDate}</TableCell>
+                      <TableCell>{treatment.endDate}</TableCell>
+                      <TableCell>{treatment.instructions}</TableCell>
+                    </TableRow>
+                  ))}
+                </TableBody>
+              </Table>
+            </div>
+          </CardContent>
+        </Card>
+
+        <Card>
+          <CardHeader>
+            <CardTitle>Analytics</CardTitle>
+            <CardDescription>Medical analytics</CardDescription>
+          </CardHeader>
+          <CardContent>
+            <ul className="space-y-2">
+              {analytics.map((analysis, index) => (
+                <li key={index} className="flex items-center justify-between">
+                  <div className="flex items-center space-x-2">
+                    <FileIcon className="h-5 w-5 text-blue-500" />
```

Comment:
please do not use hardcoded colors.  Make light and dark color variants in the css files.